meta/runtime.yml

```diff
@@ -98,6 +98,10 @@ plugin_routing:
       redirect: vyos.vyos.vyos
     ping:
       redirect: vyos.vyos.vyos
+    vyos_prefix_lists:
```

Comment:
please add in modules section also

Reply:
I tried adding these in modules section as well but adding these is modules, doesn't redirect properly.

Reply:
you only need to redirect prefix_lists
```
prefix_lists:
      redirect: vyos.vyos.vyos_prefix_lists
```

Reply:
Thanks. Updated. Works fine now.